docs/nightly/en/user-guide/ingest-data/for-observerbility/opentelemetry.md

```diff
@@ -0,0 +1,82 @@
+# OpenTelemetry Protocol (OTLP)
+
+[OpenTelemetry](https://opentelemetry.io/) is a vendor-neutral open-source observability framework for instrumenting, generating, collecting, and exporting telemetry data such as traces, metrics, logs. The OpenTelemetry Protocol (OTLP) defines the encoding, transport, and delivery mechanism of telemetry data between telemetry sources, intermediate processes such as collectors and telemetry backends.
+
```

Comment:
OTLP is the underlying protocol. We also need to cover how to configure tools from OpenTelemetry ecosystem to ingest data. But this can be authored in another pr.